src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -639,8 +677,7 @@
                     "Validators": [
                         {
                             "ValidatorType": "Range",
-                            "Configuration": 
-                            {
+                            "Configuration": {
```

Comment:
nit: there's a lot of whitespace changes. can you remove them all?

Reply:
This is just how VS auto formats when you make edits. I don't really want to fight with the format back and forth.